server/models/snippet.js

```diff
@@ -1,30 +1,62 @@
-const mongoose = require("mongoose");
+const mongoose = require('mongoose');
 
 const snippetSchema = new mongoose.Schema({
-	code: {
-		type: String,
-		required: true,
-		
-	},
-	author: {
-		type: mongoose.Schema.Types.ObjectId,
-		required: true,
-		ref: "User",
-	},
-	reviewer: {
-		type: mongoose.Schema.Types.ObjectId,
-		ref: "User",
-	},
-	title: {
-		type: String,
-		required: true,
-		default: "Unknown",
-	},
-	language: {
-		type: String,
-		required: true,
-	},
+  code: {
+    type: String,
+    required: true,
+  },
+  type: {
+    type: String,
+    required: true,
+    enum: ['request', 'receive'],
+  },
+  status: {
+    type: String,
+    enum: ['pending', 'accepted', 'completed'],
+    default: 'pending',
```

Comment:
As discussed in today stand-up, we should change `accepted `to `in review`.

Reply:
I changed it